detekt/detekt-config.yml

```diff
@@ -119,6 +119,7 @@ complexity:
     ignoreDefaultParameters: true
     ignoreDataClasses: true
     ignoreAnnotated: []
+    excludes: ['**/*Nonce.kt']
```

Comment:
curious: Does converting the nonces to data classes prevent lint errors?

Reply:
great catch! yep, converting them to data classes removes the detekt error. although, `CardNonce` needs the suppress annotation since it's an open class and can't also be defined as a data class. i just pushed an update for this!

Reply:
Nice! We may be able to convert CardNonce one day to a data class. The `open` modifiers can be removed also, the only reason we added them in the past is to prevent breaking changes since Java classes are non-final by default.